server/events/plan_command_runner.go

```diff
@@ -26,6 +26,7 @@ func NewPlanCommandRunner(
 	lockingLocker locking.Locker,
 	discardApprovalOnPlan bool,
 	pullReqStatusFetcher vcs.PullReqStatusFetcher,
+	SetAtlantisApplyCheckSuccessfulIfNoChanges bool,
```

Comment:
Can we add a test for this in server/events/plan_command_runner_test.go ?

Reply:
@nitrocode I wrote some tests in `plan_command_runner_test.go`. Thanks for your review.